src/parquet/encodings/plain-encoding.h

```diff
@@ -52,6 +52,19 @@ class PlainDecoder : public Decoder {
     return GetValues(buffer, max_values, sizeof(int64_t));
   }
 
+  virtual int GetInt96(Int96* buffer, int max_values) {
+    max_values = std::min(max_values, num_values_);
+    int size = 3*sizeof(uint32_t);//Struct Int96 could be padded
```

Comment:
Your statement on performance got me intrigued. After a bit of googling, people seem to have mixed reviews. Performance will depend upon other factors as well like caching and SSE/AVX. I will disable padding for Int96 to simplify and avoid any specialization of templates. We can look into this later when we deep dive into performance.